fighthealthinsurance/utils.py

```
@@ -384,15 +395,38 @@
+            max_extra_time_for_optional,
+            time_remaining_before_timeout,
+        )
+        logger.debug(
+            f"Waiting for optional tasks to finish for {remaining_seconds} seconds"
+        )
+        # Wait for optional tasks to finish with a timeout
+        for task in asyncio.as_completed(optional_tasks, timeout=remaining_seconds):
+            optional_result: T = await task
+            # Yield each result immediately for streaming
+            yield optional_result
+    except asyncio.TimeoutError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)